config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  get 'setup/index'
```

Comment:
I would stick with [resourceful routing](https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default) for this, mainly because that's how we've managed routes elsewhere.

(assuming we renamed from `setup` to `setup_instructions`)
```suggestion
  resources :setup_instructions, only %i[Index]
```